package-lock.json

```diff
@@ -0,0 +1,4081 @@
+{
```

Comment:
👍 Although I can see the value of a lock file in a company shipping a customer-facing app, I don't think it is necessary for libraries and their dependencies.